src/index.ts

```diff
@@ -33,20 +31,32 @@ export default function useSound(
   };
 
   // We want to lazy-load Howler, since sounds can't play on load anyway.
-  useOnMount(() => {
-    import('howler').then(mod => {
-      HowlConstructor.current = mod.Howl;
-
-      const sound = new HowlConstructor.current({
-        src: [url],
-        volume,
-        onload: handleLoad,
-        ...delegated,
-      });
-
-      setSound(sound);
-    });
-  });
+  useEffect(() => {
```

Comment:
Can't this still be done using the `useOnMount` helper? It should work exactly the same way